Pocket/pocket-ios
# fix(tags): sorting all tags alphabetically

## Summary

Tags were not holding their sort order within coredata, likely due to duplicate tags merging in coredata. Unfortunately the API also does not have a timestamp or other property to order tags on a SavedItem. 

So instead we will manually sort all tags via `.sorted` when we display them to order them alphabetically by name.

This is also the cause of a number of test failures because tags would show in different order on Item cells in the list. Without this change, you can see this happen live on `test_tappingTagLabel_showsTagFilter` when run repeatedly.

## Test Steps
* Run `test_tappingTagLabel_showsTagFilter` and ensure tags show on Item 2 in the same order every time. (try without this pr to see it in different orders)

## PR Checklist:

## Screenshots
